### PR TITLE
 TEAM-C-Premium-Section-(UI)

### PR DESCRIPTION
**Pull request by BunyaminOzturk**

### CODE REVIEW

**Comment by BunyaminOzturk** [comment]
I added the desired premium ui feature according to the task

File: app/src/main/res/drawable/premium_frame.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <!-- frame is rectangle -->
```
Comment (oliver-basoglu) [inline app/src/main/res/drawable/premium_frame.xml]: minor: Unnessary comment.

File: app/src/main/res/layout/fragment_profile.xml [hunk 2]
```diff
@@ -94,11 +94,77 @@
         app:layout_constraintTop_toBottomOf="@+id/displayNameTv"
         tools:text="example@example.com" />
 
+    <LinearLayout
```
Comment (oliver-basoglu) [inline app/src/main/res/layout/fragment_profile.xml:94]: minor: Try to use ConstraintLayout.

File: app/src/main/res/layout/fragment_profile.xml [hunk 3]
```diff
@@ -215,10 +281,10 @@
 
                 <ImageView
                     android:layout_width="28dp"
-                    app:tint="@color/text_color"
-
                     android:layout_height="28dp"
-                    android:src="@drawable/shield_done" />
+
```
Comment (oliver-basoglu) [inline app/src/main/res/layout/fragment_profile.xml]: remove the space

**Comment by kilicma** [comment]
good job

**Review by Feyzarkt — COMMENTED** [review]
Good work🙌 You may try to use ConstraintLayout in fragment_profile.xml

**Review by kilicma — COMMENTED** [review]
good job

File: app/src/main/res/layout/fragment_profile.xml (hunk 3 above)
Comment (BunyaminOzturk) [reply app/src/main/res/layout/fragment_profile.xml]: Thank you for your advice

**Review by cihancelik7 — APPROVED** [review]

**Review by kilicma — APPROVED** [review]

**Pull request merged**